include/libopencm3/swm050/gpio.h

```diff
@@ -47,52 +44,74 @@
 #define GPIO7				(1 << 7)
 #define GPIO8				(1 << 8)
 #define GPIO9				(1 << 9)
-#define GPIO_ALL			0x3ff
-/**@}*/
+#define GPIO_ALL			(0x3ff)
+/*@}*/
 
 /* GPIO direction definitions */
 /** @defgroup gpio_dir GPIO Pin Direction
 @{*/
-#define GPIO_INPUT			0x0
-#define GPIO_OUTPUT			0x1
-/**@}*/
+#define GPIO_INPUT			(0x0)
+#define GPIO_OUTPUT			(0x1)
+/*@}*/
+
+/* GPIO polarity definitions */
+/** @defgroup gpio_pol GPIO Polarity
+@{*/
+#define GPIO_LOW			(0x0)
+#define GPIO_HIGH			(0x1)
```

Comment:
you should make this something like GPIO_POL_ or GPIO_POLARITY_ I would think, and same for the irq triggers, so you know which one you're using where.  There's a hacking file that says how to name defines.

Reply:
Fixed.